pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/FunctionsImpl.java

```diff
@@ -70,11 +71,13 @@
 public class FunctionsImpl extends ComponentResource implements Functions {
 
     private final WebTarget functions;
+    private final WebTarget worker;
     private final AsyncHttpClient asyncHttpClient;
 
     public FunctionsImpl(WebTarget web, Authentication auth, AsyncHttpClient asyncHttpClient, long readTimeoutMs) {
         super(auth, readTimeoutMs);
         this.functions = web.path("/admin/v3/functions");
+        this.worker = web.path("/admin/v2/worker");
```

Comment:
Should `available-functions` be a command of `pulsar-admin workers` instead ?

Reply:
+1 for `pulsar-admin functions`. However it's a bit weird that the rest endpoint is under "/worker"

Reply:
This is because we have the list of connectors under `/worker`.
Now we also have the list of sinks under `/sinks` and the list of sources under `/sources`.
So I think we can indeed have the list of functions under `/functions`